libraries/lib-module-manager/ModuleManager.cpp

```diff
@@ -479,34 +475,34 @@ void ModuleManager::InitializeBuiltins()
    }
 }
 
-void ModuleInterfaceDeleter::operator() (ModuleInterface *pInterface) const
+void PluginProviderDeleter::operator() (PluginProvider *pInterface) const
 {
    if (pInterface)
    {
       pInterface->Terminate();
-      std::unique_ptr < ModuleInterface > { pInterface }; // DELETE it
+      std::unique_ptr < PluginProvider > { pInterface }; // DELETE it
    }
 }
 
 bool ModuleManager::RegisterEffectPlugin(const PluginID & providerID, const PluginPath & path, TranslatableString &errMsg)
 {
    errMsg = {};
-   if (mDynModules.find(providerID) == mDynModules.end())
+   if (mProviders.find(providerID) == mProviders.end())
    {
       return false;
    }
 
-   auto nFound = mDynModules[providerID]->DiscoverPluginsAtPath(path, errMsg, PluginManagerInterface::DefaultRegistrationCallback);
+   auto nFound = mProviders[providerID]->DiscoverPluginsAtPath(path, errMsg, PluginManagerInterface::DefaultRegistrationCallback);
 
    return nFound > 0;
 }
 
-ModuleInterface *ModuleManager::CreateProviderInstance(const PluginID & providerID,
+PluginProvider *ModuleManager::CreateProviderInstance(const PluginID & providerID,
                                                       const PluginPath & path)
 {
-   if (path.empty() && mDynModules.find(providerID) != mDynModules.end())
+   if (path.empty() && mProviders.find(providerID) != mProviders.end())
```

Comment:
@Paul-Licameli Not relates to this PR, but I wonder why do we need `path` variable here?

Reply:
Good question.  I don't remember why.  It looked strange to me too.